### ISSUE TITLE
Problem running SAVER on large dataset

### ISSUE THREAD

**Issue by RomiGoldner** [mohuangx/SAVER#32]
Hello,
I have a large dataset with 7685 cell and 18,000 genes that I was to use SAVER on (for imputation).
I tried running it with 12, 6, 4, and 2 cores (setting ncore argument) and it doesn't seem to work. 
I also tries to split the genes to subgroups and combine using the saver combine but that didn't work either. 

Is there a way you can assist me? Or have any recommendations?
Thanks in advance!

**Comment by mohuangx** [comment]
Hi,

Can you provide the function call and the error message that you get?

Thanks!

**Comment by RomiGoldner** [comment]
The function is the general "saver" function.
It starts the computation process and after computing the lambda coefficients, when it predicts the "remaining genes" RStudio crashes. There is no specific error.

**Comment by mohuangx** [comment]
And the same thing happened when you tried to split the genes into subgroups?

It sounds to me like a memory issue. Can you try running the example code in the `saver` help file to see if that works? Or maybe try running it on a subset of 1000 genes and 100 cells.

**Comment by RomiGoldner** [comment]
Yes, it happened when I tried to split it into subgroups of about 2500 genes.
I will try smaller groups and will update. Thank you!
